### DISCUSSION TITLE
turtlebot4_bringup not found

### DISCUSSION THREAD

**Discussion by Akalpita30** [iRobotEducation/create3_docs#437] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Ubuntu version 22.04

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.2

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

turtlebot4_bringup not found: tried installing with "sudo apt update && sudo apt upgrade" still shows the same error.  Moreover "ros2 service call /stop_motor std_srvs/srv/Empty {}" doesn't stop the motor.

**Comment by shamlian** [comment]
Sounds like you might be setting up a Turtlebot 4. ClearPath's TurtleBot 4 is a derivative product of iRobot's Create 3 robot, with its own separate configuration. Please direct any questions regarding TurtleBot 4 [here](https://github.com/turtlebot/turtlebot4/issues/).
